util/getNodeDownloadUrl.ts

```diff
@@ -59,6 +59,13 @@ export const getNodeDownloadUrl = (
       // Since the x86 platform is not officially supported, returns the x64
       // link as the default value.
       return `${baseURL}/node-${versionWithPrefix}-linux-x64.tar.xz`;
+    case 'AIX':
+      // Prepares a downloadable Node.js link for AIX
+      if (typeof bitness === 'string') {
+        return `${baseURL}/node-${versionWithPrefix}-aix-${bitness}.tar.gz`;
```

Comment:
What sort of bitnesses does this support?

Reply:
What happens if the browser is unable to detect bitness? If this is always x64, shouldn't we remove the bitness variable and just set it to x64?

Reply:
cc @mhdawson

Reply:
It's not x64, it's ppc64, e.g. https://nodejs.org/dist/v6.7.0/node-v6.7.0-aix-ppc64.tar.gz.
I agree though that we should just always set rather than attempt any sort of detection.

Reply:
Yes, detection doesn't make sense here if there's only one option IMO. Even more because if the bitness detection fails, it will fallback to x86 I think? cc @canerakdas